js/popup.js

```diff
@@ -0,0 +1,180 @@
+      'Codepen',
+    ],
+    liveVersionLink: 'https://dd-obua.github.io/Mic-Portfolio/',
+    sourceLink: 'https://github.com/dd-obua/Mic-Portfolio',
+  },
+  {
+    name1: 'Multi-Post Stories',
+    name2: 'Gain+Glory',
+    description1:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.",
+    description2:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.",
+    featuredImage: './img/featured/6.jpg',
+    popupTitle: 'Keeping track of hundreds of components',
+    technologies: [
+      'Ruby on Rails',
+      'CSS',
+      'JavaScript',
+      'HTML',
+      'Bootstrap',
+      'Terminal',
+      'Codepen',
+    ],
+    liveVersionLink: 'https://dd-obua.github.io/Mic-Portfolio/',
+    sourceLink: 'https://github.com/dd-obua/Mic-Portfolio',
+  },
+];
+
+cards.forEach((card, index) => {
+  const cardElementHtml = `
+    <div class="img-container">
+      <img src="${card.featuredImage}" alt="Featured image" class="featured-img">
+    </div>
+    <div class="information">
+      <h3 class="works__stories--heading">${card.name1} <span class="gain-glory">${card.name2}</span></h3>
+      <ul class='works__categories'>
+        <li class='works__skill'>${card.technologies[0]}</li>
+        <li class='works__skill'>${card.technologies[1]}</li>
+        <li class='works__skill'>${card.technologies[2]}</li>
+        <li class='works__skill'>${card.technologies[3]}</li>
+      </ul>
+      <a href="#" class="link see-project" data-index="${index}">See Project</a>
+    </div>`;
+
+  const cardElement = document.createElement('article');
+  cardElement.className = 'works__card';
+  cardElement.innerHTML = cardElementHtml;
+  worksCards.appendChild(cardElement);
+});
+
+const mobilePopup = document.querySelector('.popup-mobile');
+const createMobilePopup = () => {
+  mobilePopup.classList.remove('take-out');
+};
+
+const desktopPopup = document.querySelector('.popup-desktop');
+const createDesktopPopup = () => {
+  desktopPopup.classList.remove('take-out');
+};
+
+const detailsLinks = document.querySelectorAll('.see-project');
+detailsLinks.forEach((link) => {
+  if (window.innerWidth >= 768) {
+    link.addEventListener('click', createDesktopPopup);
+  } else link.addEventListener('click', createMobilePopup);
+});
+
+const cancelMobilePopup = document.querySelector('.close-mobile-popup');
+const cancelDesktopPopup = document.querySelector('.close-desktop-popup');
+
+cancelMobilePopup.addEventListener('click', () => {
+  mobilePopup.classList.add('take-out');
+});
+
+cancelDesktopPopup.closest('.img-div').addEventListener('click', () => {
+  desktopPopup.classList.add('take-out');
+});
```

Comment:
Good job so far 👏🏽. Please note that in the original design, when the pop-up appears the background is blurred but that is not the case with your design. Kindly fix.
Your Design
![Screenshot from 2023-07-29 15-19-39](https://github.com/dd-obua/Mic-Portfolio/assets/44624138/8e54bb6c-4f7e-4085-9be3-dc019cb929d6)
Figma design
![Screenshot from 2023-07-29 15-12-54](https://github.com/dd-obua/Mic-Portfolio/assets/44624138/f0ae6ad7-73c5-4821-8821-eade5b382017)

Reply:
Please note that on the mobile screen when I click the see more button, the pop-up appears below the page 😢. Kindly fix this. 👍🏽
[Screencast from 29-07-2023 15:10:35.webm](https://github.com/dd-obua/Mic-Portfolio/assets/44624138/f88353f8-27ec-4fe7-93d0-77a0bfd113f9)